x-pack/plugins/secops/public/components/app_settings/app_settings_popover.test.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mount } from 'enzyme';
+import { noop } from 'lodash/fp';
+import * as React from 'react';
+
+import { AppSettingsPopover } from './app_settings_popover';
+
+describe('AppSettingsPopover', () => {
+  describe('rendering', () => {
+    test('it renders a settings gear icon', () => {
+      const wrapper = mount(
+        <AppSettingsPopover onClick={noop} onClose={noop} showPopover={false} />
+      );
+
+      expect(wrapper.find('[data-test-subj="gear"]').length).toBeTruthy();
```

Comment:
Optional, you can use exists here if you want to just test that it is on the DOM:
https://airbnb.io/enzyme/docs/api/ReactWrapper/exists.html

Might be more semantic for the reader. But this is fine as is.